{{cookiecutter.project_dashed}}-src/ansible/provision.yaml

```diff
@@ -1,6 +1,8 @@
 {% raw %}
 - name: web app provision
   hosts: webservers
+  become: yes
+  gather_facts: yes
```

Comment:
I've avoided gathering facts since it's slow and we usually have a pretty homogeneous environment for any given app.  Whatever variables are needed, I just put in the `all` file.